src/gui/optionsdlg.cpp

```diff
@@ -357,6 +353,25 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     m_ui->advPageLayout->addWidget(advancedSettings);
     connect(advancedSettings, SIGNAL(settingsChanged()), this, SLOT(enableApplyButton()));
 
+    m_ui->textFileLogPath->setDialogCaption(tr("Choose a save directory"));
```

Comment:
Nope, I do not export the widget to Designer completely. Probably, with only a single custom widget in the app it does not make sense.